src/app/containers/ATIAnalytics/params/article/buildParams.test.js

```diff
@@ -115,6 +115,7 @@ describe('buildParams', () => {
           'x12=[1970-01-01T00:00:00.000Z]',
           'x13=[thing+label+1~thing+label+2]',
           'x14=[thing+id+1~thing+id+2]',
+          'ref=originpreviousPath',
```

Comment:
Reading through the issue again, it seems we have these specific test cases:

Case 1)
Given there is a value we are putting in the x6 field
When I visit any Simorgh page, 
Then I should see the ATI request with a ref query parameter
And the value should not be wrapped in square brackets
And the value should be the escaped version of the x6 value *
And it must always be the final one in the ATI call **

I don't think we have a unit test for this last case (eg it would be easy for someone to add another ATI query parameters at the end of the file later on and this requirement would not be kept. 
Also, I'm not quite sure if I understood requirement on line * correctly. It would be good to update the test value to include characters that need escaping. Eg the referrer could be https://www.example.com, so we can see in the unit tests that it's being encoded correctly. 

Case 2)
Given there is no value that we are putting in the x6 field
Then do not append a ref value
And do not leave an empty value, e.g. ref=

I don't believe we have a unit test for this scenario? Unless I've missed it.

Reply:
On case 2, it is (from my POV) equally acceptable to have no `ref` or to have an empty value `ref=`

Reply:
@jimjohnsonrollings Okay, thanks. That wasn't too clear to me when reading the issue, which had this line:
`Then do not append a ref value OR leave an empty value, e.g. ref=`
Thanks for clarifying.

Reply:
You are right, that wasn't a very clear way I wrote that.

Reply:
Pulling this back to In Progress to add tests for **, *, and `Given there is no value that we are putting in the x6 field, Then we do not append a ref value`

Reply:
Added the above mentioned test cases [here](https://github.com/bbc/simorgh/pull/6412/files/ec3beed20ad10458fb79a9ed576eb99b7e7fa584#diff-59330db613648920566256700c5d6c8d)